src/index.js

```diff
@@ -120,11 +120,11 @@ export function unionOf(schema, options) {
 export { EntitySchema as Schema };
 
 export function normalize(obj, schema, options = {}) {
-  if (!isObject(obj) && !Array.isArray(obj)) {
```

Comment:
I think you want to keep the first check: it is specifically whitelisting arrays.
The second check can probably be removed. Note that the first one has `!` but the other doesn't.